bcipy/display/paradigm/vep/layout.py

```diff
@@ -165,9 +165,35 @@ def positions(self) -> List[Tuple]:
         left = right_of(layout.left, width / 2)
         right = left_of(layout.right, width / 2)
 
-        positions = [(left, top), (right, top), (left, bottom),
-                     (right, bottom)]
-        if self.num_boxes == 6:
-            positions += [(layout.horizontal_middle, top),
-                          (layout.horizontal_middle, bottom)]
-        return positions
+        if self.num_boxes == 4:
```

Comment:
We might consider making these a type (4by4VEP, etc) to help set other properties as well.

Reply:
In this context the num_boxes refers to the number of display areas where a symbol can be sorted. The VEPStim is where the number of squares per checkerboard is defined. If we only ever want to fix this at 6 display areas, I can remove the conditional logic. The other idea here is that if we want a different layout we can provide another class with the same interface for generating these positions.

Reply:
I like that last idea! Unless we are going to test 4 vs 6 in the short term, I think it would be easier to maintain cutting it out.